src/node.ts

```diff
@@ -4,6 +4,7 @@ import sourceMapSupport from 'source-map-support'
 import { transformSync } from 'esbuild'
 import { addHook } from 'pirates'
 import { getOptions } from './options'
+import { FILE } from 'dns'
```

Comment:
@antfu @egoist I don't remember adding this line. Must have autoimported it by error. I think it's of no use.

Reply:
Could you clean it up so I can request for another build update? Thanks.